core/src/test/groovy/com/novoda/gradle/release/TestConfiguration.groovy

```diff
@@ -73,9 +81,21 @@ class TestBintrayRelease {
         Node packagingNode = nodes[5]
         assertThat(packagingNode.name().localPart).isEqualTo "packaging"
         assertThat(packagingNode.value()[0]).isEqualTo "aar"
+    }
+
+    @Test
+    public void testBintrayUploadTask() {
+        Task task = project.getTasks().findByPath(":bintrayUpload")
+        assertThat(task).isNotNull()
+
+        BintrayUploadTask uploadTask = task as BintrayUploadTask
+        uploadTask.user = "user"
+        uploadTask.apiKey = "apiKey"
 
-        // uploadTask verification here
+        uploadTask.repoName = "maven"
+        uploadTask.packageName = "test"
+        uploadTask.executeWithoutThrowingTaskFailure()
 
-        project.buildDir.deleteDir()
+        assertThat(uploadTask.dryRun).isFalse()
```

Comment:
I wasn't able to get this configuration up and running for some reason,
both before and after my changes. Not sure if it has something to do with
the dummy fixture directory being too lean or something else :/
On Tue, Aug 25, 2015 at 6:41 AM Friedger Müffke <notifications@github.com>
wrote:

> In core/src/test/groovy/com/novoda/gradle/release/TestConfiguration.groovy
> <https://github.com/novoda/bintray-release/pull/67#discussion_r37865576>:
>
> >
> > -        project.buildDir.deleteDir()
> > +        assertThat(uploadTask.dryRun).isFalse()
>
> This should assert for isTrue. However, the task is not configured (see
> comment above).
>
> @devisnik <https://github.com/devisnik> @hzsweers
> <https://github.com/hzsweers> Any ideas how to improve that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/bintray-release/pull/67/files#r37865576>.
>